packages/ui/primitives/theme-switcher.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+
+import { motion } from 'framer-motion';
+import { Monitor, MoonStar, Sun } from 'lucide-react';
+
+interface ThemeSwitcherProps {
```

Comment:
We prefer types over interfaces to avoid mixing the two

Reply:
Ohh that makes sense why I didn't see many usages interfaces, I will fix that one up as well :)